packages/lib/versionInfo.test.ts

```diff
@@ -0,0 +1,30 @@
+import { _ } from './locale';
+import { getPluginsArr } from './versionInfo';
+import PluginService from '@joplin/lib/services/plugins/PluginService';
+
+const plugins = getPluginsArr(PluginService.instance().plugins);
+
+describe('versionInfo', function() {
+
+	it('should return an array', (()=>{
```

Comment:
If you add return type annotation to `getPluginsArr` then this test becomes useless

Reply:
I will remove the test and add types to the function